src/util/parse_options.cpp

```diff
@@ -37,7 +51,7 @@ void parse_options_baset::help()
 
 void parse_options_baset::usage_error()
 {
-  std::cerr << "Usage error!\n\n";
+  log.error() << "Usage error!\n\n";
```

Comment:
I believe that needs a `<< messaget::eom`

Reply:
Point.

Reply:
Fixed.